packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -166,8 +152,16 @@ export function convertWireToOsdkObjects<
       });
     }
 
-    const proto = getPrototype(client.ontology, obj.__apiName);
-    const converter = getConverter(client.ontology, obj.__apiName);
+    const objectDef = await client.ontology.provider
```

Comment:
Do we actually want to iterate through `objs` ahead of time so we can request definitions in parallel?